docs/development/CUSTOM_RULES.md

```diff
@@ -46,27 +46,30 @@ export const rule = createRule({
   create(context) {
     return {
       FunctionDeclaration(node) {
-        if (/^[a-z]/.test(node.id.name)) {
+   if(node.id!=null){
+          if (/^[a-z]/.test(node.id.name)) {
           context.report({
             messageId: 'uppercase',
             node: node.id,
           });
         }
+   }
       },
     };
   },
+  name: 'rule-creator',
```

Comment:
This `name` is the name of the rule, which is typically 1-3 words summarizing what the rule enforces.

Maybe... `uppercase-first-declarations`? Something like that?

Reply:
ya sure